<launch>
	<node pkg="ackermann2actuator" type="measure_velocity_node" name="measure_velocity_node" output="screen">
	    <rosparam file="$(find ackermann2actuator)/config/ackermann2actuator.yaml" command="load" />
		<param name="input_measurement_series" value="/home/philipp/tmp/measurements/test.csv" />
	    <param name="output_measurement_series" value="/home/philipp/tmp/measurements/test.csv" />
	    <remap from="velocity_actuator" to="motor" />
	    <remap from="steering_actuator" to="servo" />
	</node>
</launch>
